doc/arch/014-region-level-sync.md

```diff
@@ -0,0 +1,47 @@
+# Region level sync
+September 2020 (accepted)
+
+_This ADR has been added retroactively in Apr 2021 to capture our switch to block-level syncing._
+
+## Context
+
+Users in large districts reported that the Simple app was running very slow, making the app near-unusable.
+The slowdown was correlated to the volume of patient data synced to the user’s phone. We realised that the amount of data being stored on the device had to be reduced for better long-term performance.
+
+Currently we sync the entire district's records to a user's phone. Some of the large districts have upto 50,000 patients, which can amount to 400-500 MB of data.
+
+A district typically has between 1-20 blocks. From trends in IHCI, we found that patients rarely move between facilities across blocks. Patients that have a BP taken in more than 1 block is around 2% or under, with the exceptions of: Sindhudurg (9.8%), Hoshiarpur (5.3%), Bathinda (3.1%).
+This means that we can sync only a block's data to the user's phone and be reasonably confident about finding patients on the app.
```

Comment:
> we found that patients rarely move between facilities across blocks

We have an average of ~600 duplicates across blocks per month in the last few months. Is this still what we would consider rare? Perhaps we should consider rewording accordingly.